src/envoy/http/service_control/client_cache_test.cc

```diff
@@ -141,7 +144,7 @@ TEST_F(ClientCacheCheckResponseTest, Sc5xxAllowed) {
   CheckError* check_error = response->mutable_check_errors()->Add();
   check_error->set_code(CheckError::NAMESPACE_LOOKUP_UNAVAILABLE);
 
-  runTest(Code::OK, response, Code::OK);
+  runTest(Code::OK, response, Code::OK, ApiKeyState::NOT_CHECKED);
```

Comment:
Why is is NOT_CHECKED?  

If there are not check_errors,  can we assume api-key is checked?  Can we assume Chemist check api-key first?

Reply:
If there are no check errors and no http network errors, then yet. The current code treats it as VERIFIED.